src/components/Comments/ReplyCommentItem.js

```diff
@@ -46,6 +55,15 @@ const ReplyCommentItem = ({
     comment.data.count_upvote - comment.data.count_downvote
   );
   const [statusVote, setStatusVote] = React.useState('');
+  const [loading, setLoading] = React.useState({
+    loadingDm: false,
+    loadingDmAnon: false
+  });
+
+  const [profile] = React.useContext(Context).profile;
+  const {createSignChat} = useCreateChat();
+  const {sendMessageDM} = useDMMessage();
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The loading state is being managed using a state object. This could potentially lead to unnecessary re-renders if not all properties of the state are used in the render function. Consider splitting this into two separate `useState` calls for `loadingDm` and `loadingDmAnon`.



<!-- This is an auto-generated comment by OSS CodeRabbit -->